lib/mountain_view/configuration.rb

```diff
@@ -1,9 +1,13 @@
 module MountainView
   class Configuration
-    attr_accessor :included_stylesheets
+    attr_accessor :included_stylesheets, :components_path
```

Comment:
As you're defining `components_path=` below, what do you think about putting `components_path` in an `attr_reader`?